src/main/kotlin/io/openfuture/api/component/solidity/CompilationResult.kt

```diff
@@ -0,0 +1,81 @@
+package io.openfuture.api.component.solidity
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Component
+import java.io.IOException
+import java.nio.file.Path
+import java.nio.file.Paths
+
```

Comment:
Why it is Component?

Reply:
Initially i used this component as dependency so i needed bean in order to use, later i removed dependency. i will remove it